File: app/src/main/java/com/example/whattodo/preferences/PreferencesActivity.java
package com.example.whattodo.preferences;

import android.os.Bundle;
import android.util.Log;

import androidx.appcompat.app.AppCompatActivity;

import com.example.whattodo.R;
import com.example.whattodo.preferences.PreferencesFragment;

public class PreferencesActivity extends AppCompatActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.shared_prefs);
        Log.i("PreferenceActivity", "onCreate");
        getFragmentManager().beginTransaction().replace(android.R.id.content,new PreferencesFragment()).commit();

    }
}

File: app/src/main/java/com/example/whattodo/preferences/PreferencesActivity.kt
package com.example.whattodo.preferences

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import com.example.whattodo.R

class PreferencesActivity : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.shared_prefs)
        Log.i("PreferenceActivity", "onCreate")
        fragmentManager.beginTransaction().replace(android.R.id.content, PreferencesFragment()).commit()
    }
}